driver/driver.cpp

```diff
@@ -63,28 +68,29 @@ Routine Description:
 
         vec2d carry = devExt->carry;
 
-        auto it = InputDataStart;
-        do {
+        for (auto it = InputDataStart; it != InputDataEnd; ++it) {
```

Comment:
i think do-while is better here

it always iterates at least once, and only once under normal conditions

Reply:
Are you 1000% sure it won't send 0 bytes? I prefer 1 cpu cycle performance drop instead of BSOD, or cursor jumping randomly.

Reply:
i don't have the exact reference on hand, but yeah i'm sure we won't ever get 0 packets

Reply:
ok, reverted